src/blog/2023/01/flowforge-1-3-0-released.md

```diff
@@ -0,0 +1,86 @@
+In FlowForge 1.3 we’ve added the ability to filter your admin logs by user or action type. We think this is a great new feature which will help admins have confidence that they will be able to review the audit logs quickly when needed. You can see a demonstration of this new feature in [the video](https://youtu.be/p0Vuy5x42Go) below.
+
+<div><iframe width="560" height="315" src="https://www.youtube.com/embed/p0Vuy5x42Go" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Improvements
+
+[Allow installation of FlowForge on devices which can't access npm](https://github.com/flowforge/flowforge-device-agent/issues/45)
+We've had feedback from customers that in some cases they want to use FlowForge devices on hardware which cannot access [Node Package Manager](https://www.npmjs.com/) (npm). In a standard configuration of Node-RED, access to npm is mandatory to run your flows. In FlowForge 1.3.0 we've added the ability for you to import all the data usually installed from npm without your devices having access to the npm service.
+
+## Bug Fixes
+
+We've fixed the following bugs in this release.
+
+- Project status UI sometimes getting stuck when restarting [#1232](https://github.com/flowforge/flowforge/issues/1232)
+
+- SSO users asked to click link in email to verify [#1543](https://github.com/flowforge/flowforge/issues/1543)
+
+- SSO users unable to edit settings [#1542](https://github.com/flowforge/flowforge/issues/1542)
+
+- SSO users not redirected to editor when signing in [#1481](https://github.com/flowforge/flowforge/issues/1481)
+
+## Contributors
+
+We'd like to thank the following for their contributions to this release:
+
+[flecoufle](https://github.com/flecoufle) for their work on [#89](https://github.com/flowforge/helm/pull/89)
+
+As an open-source project, we welcome community involvement in what we're building.
+If you're interested in contributing, checkout our [guide in the docs](https://flowforge.com/docs/contribute/).
+
+### Try it out
+
+We're confident you can have self managed FlowForge running locally in under 30 minutes.
+You can install our [local build](https://flowforge.com/docs/install/local/), use [Docker](https://flowforge.com/docs/install/docker/), or [Kubernetes](https://flowforge.com/docs/install/kubernetes/).
+
+If you'd rather use our hosted offering: [Sign up for FlowForge Cloud](https://app.flowforge.com/account/create) to get your first project free for a month.
+
+### Upgrading FlowForge
+
+[FlowForge Cloud](https://app.flowforge.com) is already running 1.3.
+
+If you installed a previous version of FlowForge and want to upgrade, our documentation provides a
+guide for [upgrading your FlowForge instance](https://flowforge.com/docs/upgrade/).
+
+### Getting help
+
+Please check FlowForge's [documentation](https://flowforge.com/docs/) as the answers to many questions are covered there.
```

Comment:
I'd call out:
- https://github.com/flowforge/flowforge/discussions
- Node-RED Forums

Let's go where the community is. Slack could be less prominent IMO

Reply:
I've added those in, I've left Slack at the bottom of the list though.